Dockerfile-rpi

```diff
@@ -0,0 +1,29 @@
+FROM resin/raspberrypi-python:3.4
```

Comment:
This should be `resin/raspberrypi2-python:3.4` if you want to build it for armv7hf. `resin/raspberrypi-python:3.4` is built on armv6.